### PR TITLE
boa: rewrite model evaluate py file.

### PR DESCRIPTION
**Pull request by Txiaozhe**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ A JavaScript application framework for machine learning and its engineering.
 
 ## Why Pipcook
 
-With the mission of enabling JavaScript engineers to utilize the power of machine learning without any prerequisites and the vision to lead front-end technical field to the intelligention. [Pipcook][] is to become the JavaScript application framework for the cross-cutting area of machine learning and front-end interaction.
+With the mission of enabling JavaScript engineers to utilize the power of machine learning without any prerequisites and the vision to lead front-end technical field to the intelligence. [Pipcook][] is to become the JavaScript application framework for the cross-cutting area of machine learning and front-end interaction.
```
Comment (yorkie) [inline docs/README.md:12]: This seems not related.

File: packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
+'use strict';
+
+import fs from 'fs';
+import readline from 'readline';
+import { zip, unzip } from 'lodash';
```
Comment (yorkie) [inline packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts:5]: How about using Python's builtin zip/unzip? See https://docs.python.org/3/library/functions.html#zip.

File: packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+export const getBayesModel = function() {
+  return MultinomialNB();
+};
+
+export const saveBayesModel = function(classifier: any, filepath: string) {
+  pickle.dump(classifier, fs.readFileSync(filepath));
+};
+
+export const loadModel = function (filepath: string) {
+  return pickle.load(fs.readFileSync(filepath));
```
Comment (yorkie) [inline packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts:159]: As @utkobe discussed with others, this needs Python's "open", because the pickle.load receive the Python's file handle, see https://docs.python.org/3/library/pickle.html#pickle.load.

File: packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts (hunk 2 above)
Comment (Txiaozhe) [reply packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts:5]: There is not 'unzip' in Python's builtin?
How to unzip, may use zip(*zip(x, y)):
```python
>>> x = [1, 2, 3]
>>> y = [4, 5, 6]
>>> zipped = zip(x, y)
>>> list(zipped)
[(1, 4), (2, 5), (3, 6)]
>>> x2, y2 = zip(*zip(x, y))
>>> x == list(x2) and y == list(y2)
True
```
'*' is useless in js.

File: docs/README.md (hunk 1 above)
Comment (Txiaozhe) [reply docs/README.md:12]: Typos are disturbing :(

File: packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts (hunk 3 above)
Comment (Txiaozhe) [reply packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts:159]: In #107 , 'open' was be replaced :(

**Comment by Txiaozhe** [comment]
This pr same as #107, so it need be labeled for 'boa'?

File: packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts (hunk 3 above)
Comment (yorkie) [reply packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts:159]: Yea, that's not working and we'll fix it later.

**Comment by yorkie** [comment]
> This pr same as #107, so it need be labeled for 'boa'?

Actually unrelated, the label "boa" might be used for boa's bug report, improvements and discussions.

**Comment by Txiaozhe** [comment]
> > This pr same as #107, so it need be labeled for 'boa'?
> 
> Actually unrelated, the label "boa" might be used for boa's bug report, improvements and discussions.

Yeah, so should #107 be labeled to 'plugin' too?

**Comment by yorkie** [comment]
Done! Thank you @Txiaozhe.

**Comment by Txiaozhe** [comment]
> Done! Thank you @Txiaozhe.

Thanks and LGTM.

File: packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts (hunk 3 above)
Comment (Txiaozhe) [reply packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts:159]: OO

File: packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts (hunk 2 above)
Comment (yorkie) [reply packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts:5]: Okay, let's use lodash edition for now :(

Comment (Txiaozhe) [reply packages/plugins/model-evaluate/bayesian-model-evaluate/src/script.ts:5]: enen :(

File: docs/README.md (hunk 1 above)
Comment (yorkie) [reply docs/README.md:12]: May I ask you to open another PR for this?

Comment (Txiaozhe) [reply docs/README.md:12]: Yeah, of cause~

**Comment by Txiaozhe** [comment]
#105
